package.json

```diff
@@ -34,7 +34,8 @@
   ],
   "scripts": {
     "start": "node ./run.js",
-    "test": "grunt lint"
+    "test": "grunt lint",
+    "postinstall": "cp libraries/libffmpegsumo.so node_modules/nw/nwjs/libffmpegsumo.so"
```

Comment:
I imagine we could get more complex with `postinstall` over time. Can we relocate it to a `postinstall.sh` with the following template:

```bash
#!/usr/bin/env bash
# Exit on first error and output comments
set -e
set -x

# Copy over libffmpegsumo.so
cp libraries/libffmpegsumo.so node_modules/nw/nwjs/libffmpegsumo.so
```